### ISSUE TITLE
Combine the Eastern (EAMO) and Western Missouri (WEMO) Regions and the newly added Arkansas Region into a single region (MOAK)

### ISSUE THREAD

**Issue by OhMcGoo** [redcross/smoke-alarm-portal#276]
Note that the new MOAK region includes nine Illinois and six Kansas counties and excludes two Arkansas and four Missouri counties. See the attached spreadsheet and regional map.
[MO-AK getasmokealarm Counties.xlsx](https://github.com/redcross/smoke-alarm-portal/files/2881643/MO-AK.getasmokealarm.Counties.xlsx)
[RCMOAR_REG.pdf](https://github.com/redcross/smoke-alarm-portal/files/2881644/RCMOAR_REG.pdf)

**Comment by kfogel** [comment]
Look at issue #277 when looking at this.  It probably makes sense to do the #277 schema changes first, so that when we combine EAMO, WEMO, and Arkansas to make the new MOAK region, the county/chapter data for MOAK is already there (and for every other region, it will be empty for now, but at least there will be a slot for that data when we get the data).

**Comment by kfogel** [comment]
Hey, @OhMcGoo: in the spreadsheet, some of the rows have overstrikes across them.  Of the overstruck rows, some of them are in IL, rather than in MO or AK.  There are also some non-overstruck rows that have IL.

Can you help me interpret this data?

**Comment by OhMcGoo** [comment]
See my comment above regarding the county makeup of the Region. Look, also, at the map. Here is a revised spreadsheet with the 'strikethrough counties' removed.
[MO-AK getasmokealarm Counties.xlsx](https://github.com/redcross/smoke-alarm-portal/files/2886403/MO-AK.getasmokealarm.Counties.xlsx)

**Comment by frankduncan** [comment]
@OhMcGoo  @kfogel  Do we want to go back through old requests that may have been in zip codes not previously handled that are part of this new region and assign them here?

**Comment by frankduncan** [comment]
As part of this issue, we're going to do all counties, chapters, and regions in the spreadsheet linked in this comment: https://github.com/redcross/smoke-alarm-portal/issues/277#issuecomment-466219685

**Comment by OhMcGoo** [comment]
@frankduncan, @kfogel: I don't think we should try to reassign requests. I think it will confuse the regions. But we will need to notify them when to expect new requests from counties that were not previously a part of the application (Arkansas).

**Comment by frankduncan** [comment]
@OhMcGoo For previous requests that were part of WEMO and EAMO, should we assign those to MOAK, or keep WEMO and EAMO regions around for archival purposes?

**Comment by OhMcGoo** [comment]
@frankduncan: Let's combine all past EAMO and WEMO requests under MOAK but I would suggest consulting with @kfogel about how to handle the request IDs (as they contain the strings EAMO and WEMO).

**Comment by frankduncan** [comment]
@OhMcGoo @kfogel I believe we should keep the IDs the same, because they may have been recorded by the requester to be referenced in future conversation.

**Comment by kfogel** [comment]
@frankduncan I'm okay with that if @OhMcGoo is.  The more I think about it, the more sane it seems.

Do we want to keep having the region code as part of the ID?  I don't really know in practice how useful it is -- it depends who's discussing requests and using the request IDs as handles in those conversations.  My under-informed guess is that it *is* useful to have the region code in there and that we should keep it, but @OhMcGoo if you have any user-side knowledge to impart, please do.

**Comment by OhMcGoo** [comment]
@kfogel, @frankduncan: I'm pro-sanity.

**Comment by kfogel** [comment]
It's a controversial position, but I'm glad you're sticking to your principles.

**Comment by frankduncan** [comment]
@OhMcGoo I noticed in this spreadsheet that there are many region codes for the Dakota.  Wanted to check that that makes sense.

**Comment by frankduncan** [comment]
@OhMcGoo In addition, there's some small descrepencies between names.  For instance, in our database it's "Northwest Region", but in the spreadsheet it's "Northwest"  Do you want me to bring our DB in line with what you have, or leave it as is?

**Comment by frankduncan** [comment]
@OhMcGoo After looking over the data, the only non Arkansas county that wasn't already in our database is from Shannon, South Dakota.  So that will get added too.

**Comment by frankduncan** [comment]
@OhMcGoo Consolidating all the comments here:

* I noticed in this spreadsheet that there are many region codes for the Dakota. Wanted to check that that makes sense.
* There's some small descrepencies between names. For instance, in our database it's "Northwest Region", but in the spreadsheet it's "Northwest" Do you want me to bring our DB in line with what you have, or leave it as is?
* The only non Arkansas county that wasn't already in our database is from Shannon, South Dakota. So that will get added too.
* The counties that are crossed out in the spreadsheet are in our database, should we remove them?

**Comment by OhMcGoo** [comment]
> @OhMcGoo Consolidating all the comments here:
> 
> * I noticed in this spreadsheet that there are many region codes for the Dakota. Wanted to check that that makes sense.
> * There's some small descrepencies between names. For instance, in our database it's "Northwest Region", but in the spreadsheet it's "Northwest" Do you want me to bring our DB in line with what you have, or leave it as is?
> * The only non Arkansas county that wasn't already in our database is from Shannon, South Dakota. So that will get added too.
> * The counties that are crossed out in the spreadsheet are in our database, should we remove them?

**Comment by frankduncan** [comment]
Thanks @OhMcGoo !  And it seems the issue is actually with my spreadsheet program adding a lot of strike throughs that aren't actually there (?).  When I load it up in Excel, it's much better.

**Comment by OhMcGoo** [comment]
See my comments in the attached PDF.
[MOAK Comments.pdf](https://github.com/redcross/smoke-alarm-portal/files/2920043/MOAK.Comments.pdf)

**Comment by OhMcGoo** [comment]
@frankduncan: The availability language is missing a "T" at the beginning of the first sentence. Can you put it back in?
<img width="1280" alt="missing t" src="https://user-images.githubusercontent.com/4147733/53753070-eeccb680-3e75-11e9-9229-5c76db679be5.png">

Thanks!

**Comment by frankduncan** [comment]
Oh ack!  Fixed!

Thanks for the catch

Frank
